feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64e4e74d0fb4f0687bf4145d.md

---
id: 64e4e74d0fb4f0687bf4145d
title: Step 5
challengeType: 0
dashedName: step-5
---

# --description--

Create a `taskData` constant and set it to an empty array. This is where all the tasks and their data (title, due date, and description) will be saved so you can track them, display them on the page, and save them to `localStorage`.

> Also, use `let` to create a `currentTask` variable and set it to an empty object. You will use it to track state when editing and discarding tasks.

Reviewer: ```suggestion
Also, use `let` to create a `currentTask` variable and set it to an empty object. You will use it to track the state when editing and discarding tasks.
```

---
id: 64e4e74d0fb4f0687bf4145d
title: Step 5
challengeType: 0
dashedName: step-5
---

# --description--

> Create a `taskData` constant and set it to an empty array. This is where all the tasks and their data (title, due date, and description) will be saved so you can track them, display them on the page, and save them to `localStorage`.

Reviewer 2: ```suggestion
Create a `taskData` constant and set it to an empty array. This array will store all the tasks along with their associated data, including title, due date, and description. This storage will enable you to keep track of tasks, display them on the page, and save them to `localStorage`.
```

> Also, use `let` to create a `currentTask` variable and set it to an empty object. You will use it to track the state when editing and discarding tasks.

Reviewer 2: ```suggestion
Use `let` to create a `currentTask` variable and set it to an empty object. This variable will be used to track the state when editing and discarding tasks.
```
